x/concentrated-liquidity/lp.go

```diff
@@ -613,3 +619,31 @@ func (k Keeper) validatePositionUpdateById(ctx sdk.Context, positionId uint64, u
 
 	return nil
 }
+
+func (k Keeper) settleFinalPoolPosition(ctx sdk.Context, poolId uint64, amount0, amount1 sdk.Int) error {
+	pool, err := k.getPoolById(ctx, poolId)
+	if err != nil {
+		return err
+	}
+
+	withdrawCoins := sdk.NewCoins(sdk.NewCoin(pool.GetToken0(), amount0.Abs()), sdk.NewCoin(pool.GetToken1(), amount1.Abs()))
```

Comment:
Is there ever a case where `amount0` or `amount1` is negative? It feels like we shouldn't be putting `.Abs()` here

Reply:
The amount should be negative in every case